context-dir/src/main.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import (
+	"dagger/test/internal/dagger"
+)
+
+type Test struct{}
+
+func (m *Test) AbsolutePath(
+	// +defaultPath="/"
+	dir *dagger.Directory,
+) *dagger.Directory {
+	return dir
+}
+
+func (m *Test) RelativePath(
+	// +defaultPath="."
+	dir *dagger.Directory,
+) *dagger.Directory {
+	return dir
+}
```

Comment:
Can we also get some sub-paths? Something like `/data` and `./data`, so we can check the differences between those as well?

Reply:
I mean, `/` will point to the root of the repo while `.` will point to the current dir so we can already do some further queries in our test to get a specific dir

like `dagger call absolute-path directory --path "context-dir"` & `dagger call relative-path directory --path "src"`

Reply:
So should I still add new method?

Reply:
I think it would be good to have a new method, and put some content there, just so we can check that the path joining logic is all correct in the tests.

Reply:
I see, alright

Reply:
Changes done